src/components/Home/Home.js

```diff
@@ -56,16 +59,17 @@ export default class Home extends React.Component {
     console.log(this.state.error);
     return (
       <div className="home">
-        <Header handleChange={this.filterMovies}/>
+        <Header handleChange={this.filterMovies} />
         <main>
-          {!this.state.filteredMovies.length && <Preview className="preview" />}
+          {!this.state.filteredMovies.length ? <Preview /> : <Preview className='closed' />}
```

Comment:
this is really smart way of hiding the preview img! conditional rendering for days!!!